### DISCUSSION TITLE
How to fix this Code Scanning issues with my CI setup

### DISCUSSION THREAD

**Discussion by DJDuque** [community/community#78491] [Code Security]
Hi all, I am setting some CI checks and there are 2 things that I haven't been able to fix.

Overall, it seems that I have been able to set-up everything to work (almost) as expected except for the following:

1. If you look at [this PR](https://github.com/ALPHA-g-Experiment/alpha-g/pull/121), why is my error duplicated? It does find the intentional error I pushed, but why is it reporting it twice?
2. I changed the name of my workflow file from `clippy.yml` to `lint.yml`, and now there is a warning that I cannot get rid of (see e.g. [this])(https://github.com/ALPHA-g-Experiment/alpha-g/runs/19478884855).

I have looked online, and apparently I should be able to get rid of this warning by "Deleting the configuration", but when I click on the "..." in the top-right corner (where this option should be available) nothing happens.

![Screenshot 2023-12-09 at 12-55-43 Code scanning tool status · ALPHA-g-Experiment_alpha-g](https://github.com/community/community/assets/59413316/9ca27fc3-cfb8-487c-a5a1-ffaa9fa6d1be)

Can these 2 issues be related? Is it possible that my warnings get duplicated because I have both configurations? How can I delete the one for the file that doesn't even exist anymore?

**Comment by ft10101** [comment]
Yes, Ensure you've completely removed the old workflow file (clippy.yml), and check triggers in the new file (lint.yml) while manually deleting the outdated workflow in GitHub Actions settings.

**Reply by DJDuque** [reply]
Hi, thanks for replying.

I have checked that the `clippy.yml` file no longer exists (see [here](https://github.com/ALPHA-g-Experiment/alpha-g/tree/main/.github/workflows)).

The new file is triggering as expected. At least that is what I assume based on [this](https://github.com/ALPHA-g-Experiment/alpha-g/actions/workflows/lint.yml). Those are all the times I would have expected this new workflow to run, and it runs as expected (except for the duplicated code scanning errors).

How do I manually delete the outdated workflow? Is it from the screenshot I added? Nothing happens when I click on the "..." in the top-right corner, and I think there is where the "Delete" option should be.

**Reply by ft10101** [reply]
To manually delete an outdated workflow:

    Go to your GitHub repository.
    Click on the "Actions" tab.
    Identify the outdated workflow.
    Click on the workflow you want to remove.
    In the top-right corner, click on the three dots (...).

If nothing happens when you click the three dots, it's possible that you may not have the necessary permissions to delete workflows. Ensure that you have the appropriate repository access.
